services/src/test/java/org/apache/druid/server/AsyncQueryForwardingServletTest.java

```diff
@@ -1063,4 +969,34 @@ public int getPort()
       return port;
     }
   }
+  interface ServletTestAction {
+    void execute(AsyncQueryForwardingServlet servlet, HttpServletRequest request, HttpServletResponse response, ObjectMapper mapper, String errorMessage) throws Exception;
+  }
+  private ArgumentCaptor<Exception> executeServletAction(ServerConfig serverConfig, String errorMessage, ServletTestAction action) throws Exception
```

Comment:
Rename the method to suggest what it is actually being used for:

```suggestion
  private ArgumentCaptor<Exception> captureExceptionHandledByServlet(ServerConfig serverConfig, String errorMessage, ServletTestAction action) throws Exception
```